nym-wallet/src/layouts/AppLayout.tsx

```diff
@@ -39,9 +39,9 @@ export const ApplicationLayout: React.FC = ({ children }) => {
             <Nav />
           </Box>
           {appVersion && (
-            <Box color="#888" ml={5} mt={8}>
+            <Typography sx={{ color: (t) => t.palette.grey[500], fontSize: 14, ml: 5, mt: 8 }}>
```

Comment:
You dont need to access the theme in this way when using `sx` prop. Try `color: `grey.500`